mirrormanager2/lib/notifications.py

```diff
@@ -39,11 +39,13 @@ def fedmsg_publish(*args, **kwargs):  # pragma: no cover
     # pylint: disable=W0703
     # Ignore message about fedmsg import
     # pylint: disable=F0401
-    try:
-        import fedmsg
-        fedmsg.publish(*args, **kwargs)
-    except Exception, err:
-        warnings.warn(str(err))
+    #try:
+    #    import fedmsg
+    #    fedmsg.publish(*args, **kwargs)
+    #except Exception, err:
+    #    warnings.warn(str(err))
+    import fedmsg
+    fedmsg.publish(*args, **kwargs)
```

Comment:
I think the changes here in ``notifications.py`` should be left out.  The original idea was to make fedmsg optional only for people who really wanted to use it.  Could you revert just this file, @Devyani-Divs?